### PR TITLE
busybox: drop apt-get

### PR DESCRIPTION
**Pull request by antonlacon**
Drop the apt-get script as command not found should be sufficient. Hopefully this is no longer needed as I assume it was put in to reduce support requests.

Attempt to mimic "sudo <command>" instead of erroring out. The steamlink payload (not the steamlink installer) at least tries to use sudo to control udevadm to setup a steam controller. I don't know if other packages have similar 'packages within packages' problems when it comes to installing something. Otherwise this could be dropped to for a "command not found" error, and any potential steamlink package could install the dummy sudo.

### CODE REVIEW

**Comment by vpeter4** [comment]
For dropping apt-get script I would ask myself first what is general way of doing things like this? Because here you would drop script but [here](https://github.com/LibreELEC/LibreELEC.tv/pull/3163#issuecomment-452418589) it is nice to make some output. So what is general way? It should be unified behaviour for all cases.

File: packages/sysutils/busybox/scripts/sudo [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
-echo -e $message
 
-exit 1
+if [ -z "${1}" ]; then
+  exit 0
+fi
+
+message="This shell is already running with root access. 'sudo' is unnecessary.\n"
+message+="\nAttempting to run the command for you. Note that any environmental\n"
```
Comment (vpeter4) [inline packages/sysutils/busybox/scripts/sudo]: environmental -> environment ?

Comment (antonlacon) [reply packages/sysutils/busybox/scripts/sudo]: Thanks, changed.

**Comment by antonlacon** [comment]
This PR is a WIP. Please ping me before merging/testing. When executed, the sudo script is trying to run some of the `message` lines as commands. I'm guessing LE's shell objects to the `+=` notation.

**Comment by vpeter4** [comment]
So this PR is another one who was not actually tested on real device? This really amazes me. Why are you even sending PR which doesn't work?

File: packages/sysutils/busybox/scripts/sudo [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
-message="$message\n "
-message="$message\n With LibreELEC you have root access by default, so you dont need 'sudo'"
-message="$message\n "
+# LibreELEC runs as root for most tasks, inculding the shell. Try to mimic the
+# most common use case of "sudo <command>".
 
-echo -e $message
 
-exit 1
+if [ -z "${1}" ]; then
```
Comment (vpeter4) [inline packages/sysutils/busybox/scripts/sudo]: Something should be written in this case? Not just die silently.

File: packages/sysutils/busybox/scripts/sudo [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
 
-message="$message\n There is no working 'sudo'."
-message="$message\n "
-message="$message\n On debian/ubuntu/all general purpose linux distributions 'sudo'"
-message="$message\n allows a permitted user to execute a command as the superuser"
-message="$message\n or another user, as specified by the security policy"
-message="$message\n "
-message="$message\n With LibreELEC you have root access by default, so you dont need 'sudo'"
-message="$message\n "
+# LibreELEC runs as root for most tasks, inculding the shell. Try to mimic the
```
Comment (vpeter4) [inline packages/sysutils/busybox/scripts/sudo]: inculding -> including

**Comment by antonlacon** [comment]
My use case for the sudo mimic is resolved. Did someone else have a use for it? Is leaving the stub alone preferred? Delete the stub altogether for a `command not found` type error?

**Comment by MilhouseVH** [comment]
I'd rather keep the stub as-is - it has IMHO more pros then cons - until we drop `root` as the default user when we can remove the stub completely.

**Comment by MilhouseVH** [comment]
Thanks but I'd like to keep this as is - the fake `apt-get` script includes useful (if a little dated) information for a first-time user.

**Pull request closed without merging**